public/app/features/trails/DataTrail.tsx

```diff
@@ -171,9 +171,14 @@ export class DataTrail extends SceneObjectBase<DataTrailState> {
   static Component = ({ model }: SceneComponentProps<DataTrail>) => {
     const { controls, topScene, history } = model.useState();
     const styles = useStyles2(getStyles);
+    const showHeaderForFirstTimeUsers = getTrailStore().recent.length < 2;
 
     return (
       <div className={styles.container}>
+        { showHeaderForFirstTimeUsers && <Stack direction="column" gap={1}>
+          <Text variant="h1">Metrics</Text>
+          <Text color="secondary">Navigate through your Prometheus-compatible metrics without writing a query</Text>
+        </Stack>}
```

Comment:
A common component could be shown after `showHeaderForFirstTimeUsers &&` that contains the same duplicated content from #81496